README.md

```diff
@@ -59,13 +62,13 @@ ARG TARGETPLATFORM
 RUN xx-info env
 ```
 
-`xx` currently contains `xx-info`, `xx-apk`, `xx-apt-get`, `xx-cc`, `xx-c++`, `xx-clang`, `xx-clang++`, `xx-go`, `xx-verify`. `xx-clang` (and its aliases) creates additional aliases, eg. `${triple}-clang`, `${triple}-pkg-config`, on first invocation or on `xx-clang --setup-target-triple` call.
+`xx` currently contains `xx-info`, `xx-apk`, `xx-apt-get`, `xx-cc`, `xx-c++`, `xx-clang`, `xx-clang++`, `xx-go`, `xx-cargo`, `xx-verify`. `xx-clang` (and its aliases) creates additional aliases, eg. `${triple}-clang`, `${triple}-pkg-config`, on first invocation or on `xx-clang --setup-target-triple` call.
 
 ## Supported targets
 
-`xx` supports building from and into Linux amd64, arm64, arm/v7, s390x, ppc64le and 386, and Alpine, Debian and Ubuntu. Risc-V is supported for Go builds and for newer distros that provide Risc-V packages like `alpine:edge` or `debian:sid`.
+`xx` supports building from and into Linux amd64, arm64, arm/v7, s390x, ppc64le and 386, and Alpine, Debian and Ubuntu. Risc-V is supported for Go and Rust builds and for newer distros that provide Risc-V packages like `alpine:edge` or `debian:sid`.
 
-Go builds that don't depend on system packages can additionally target MacOS and Windows on all architectures. C/C++/CGo builds are supported for MacOS targets when an external SDK image is provided.
+Go and Rust builds that don't depend on system packages can additionally target MacOS and Windows on all architectures. C/C++/CGo builds are supported for MacOS targets when an external SDK image is provided.
```

Comment:
Rust works without external SDK?

Reply:
Ok it needs the SDK:

```
#31 57.32 not ok 9 darwin-amd64-hellocargo-rustup
#31 57.32 # (from function `assert_success' in file bats-assert/src/assert.bash, line 114,
#31 57.32 #  from function `testHelloCargo' in file test-cargo.bats, line 32,
#31 57.32 #  from function `testHelloCargoRustup' in file test-cargo.bats, line 44,
#31 57.32 #  in test file test-cargo.bats, line 93)
#31 57.32 #   `testHelloCargoRustup' failed
#31 57.32 #
#31 57.32 # -- command failed --
#31 57.32 # status : 101
#31 57.32 # output (14 lines):
#31 57.32 #      Compiling hello_cargo v0.1.0 (/work/fixtures/hello_cargo)
#31 57.32 #        Running `rustc --crate-name hello_cargo --edition=2018 src/main.rs --error-format=json --json=diagnostic-rendered-ansi,artifacts,future-incompat --crate-type bin --emit=dep-info,link -C opt-level=3 -C embed-bitcode=no -C metadata=849bb93aad9942c1 -C extra-filename=-849bb93aad9942c1 --out-dir /tmp/cargobuild/x86_64-apple-darwin/release/deps --target x86_64-apple-darwin -C linker=x86_64-apple-darwin-clang -L dependency=/tmp/cargobuild/x86_64-apple-darwin/release/deps -L dependency=/tmp/cargobuild/release/deps`
#31 57.32 #   error: linking with `x86_64-apple-darwin-clang` failed: exit status: 1
#31 57.32 #     |
#31 57.32 #     = note: "x86_64-apple-darwin-clang" "-m64" "-arch" "x86_64" "/tmp/rustczNIo9J/symbols.o" "/tmp/cargobuild/x86_64-apple-darwin/release/deps/hello_cargo-849bb93aad9942c1.hello_cargo.531a99dd-cgu.0.rcgu.o" "/tmp/cargobuild/x86_64-apple-darwin/release/deps/hello_cargo-849bb93aad9942c1.hello_cargo.531a99dd-cgu.1.rcgu.o" "/tmp/cargobuild/x86_64-apple-darwin/release/deps/hello_cargo-849bb93aad9942c1.hello_cargo.531a99dd-cgu.2.rcgu.o" "/tmp/cargobuild/x86_64-apple-darwin/release/deps/hello_cargo-849bb93aad9942c1.2k37f7clxoqfop0d.rcgu.o" "-L" "/tmp/cargobuild/x86_64-apple-darwin/release/deps" "-L" "/tmp/cargobuild/release/deps" "-L" "/root/.rustup/toolchains/stable-x86_64-unknown-linux-musl/lib/rustlib/x86_64-apple-darwin/lib" "/root/.rustup/toolchains/stable-x86_64-unknown-linux-musl/lib/rustlib/x86_64-apple-darwin/lib/libstd-0f7ee384278ce82b.rlib" "/root/.rustup/toolchains/stable-x86_64-unknown-linux-musl/lib/rustlib/x86_64-apple-darwin/lib/libpanic_unwind-6023318e4257fdb3.rlib" "/root/.rustup/toolchains/stable-x86_64-unknown-linux-musl/lib/rustlib/x86_64-apple-darwin/lib/libobject-50ed95d28fda9497.rlib" "/root/.rustup/toolchains/stable-x86_64-unknown-linux-musl/lib/rustlib/x86_64-apple-darwin/lib/libmemchr-114781e2905bc242.rlib" "/root/.rustup/toolchains/stable-x86_64-unknown-linux-musl/lib/rustlib/x86_64-apple-darwin/lib/libaddr2line-841a5df74cbbcf8e.rlib" "/root/.rustup/toolchains/stable-x86_64-unknown-linux-musl/lib/rustlib/x86_64-apple-darwin/lib/libgimli-9b35810dd2e8e276.rlib" "/root/.rustup/toolchains/stable-x86_64-unknown-linux-musl/lib/rustlib/x86_64-apple-darwin/lib/librustc_demangle-d44decaafa04c51d.rlib" "/root/.rustup/toolchains/stable-x86_64-unknown-linux-musl/lib/rustlib/x86_64-apple-darwin/lib/libstd_detect-aa335e35e4a7724c.rlib" "/root/.rustup/toolchains/stable-x86_64-unknown-linux-musl/lib/rustlib/x86_64-apple-darwin/lib/libhashbrown-7ac72202be300078.rlib" "/root/.rustup/toolchains/stable-x86_64-unknown-linux-musl/lib/rustlib/x86_64-apple-darwin/lib/libminiz_oxide-2930c6f21f36f92f.rlib" "/root/.rustup/toolchains/stable-x86_64-unknown-linux-musl/lib/rustlib/x86_64-apple-darwin/lib/libadler-654445a53da668f3.rlib" "/root/.rustup/toolchains/stable-x86_64-unknown-linux-musl/lib/rustlib/x86_64-apple-darwin/lib/librustc_std_workspace_alloc-ce034a3eed8d4113.rlib" "/root/.rustup/toolchains/stable-x86_64-unknown-linux-musl/lib/rustlib/x86_64-apple-darwin/lib/libunwind-51412ab8efb0f481.rlib" "/root/.rustup/toolchains/stable-x86_64-unknown-linux-musl/lib/rustlib/x86_64-apple-darwin/lib/libcfg_if-1c20aac4d9e33893.rlib" "/root/.rustup/toolchains/stable-x86_64-unknown-linux-musl/lib/rustlib/x86_64-apple-darwin/lib/liblibc-5559092a2ede5191.rlib" "/root/.rustup/toolchains/stable-x86_64-unknown-linux-musl/lib/rustlib/x86_64-apple-darwin/lib/liballoc-05250b6a4768a099.rlib" "/root/.rustup/toolchains/stable-x86_64-unknown-linux-musl/lib/rustlib/x86_64-apple-darwin/lib/librustc_std_workspace_core-10f98b32877a2067.rlib" "/root/.rustup/toolchains/stable-x86_64-unknown-linux-musl/lib/rustlib/x86_64-apple-darwin/lib/libcore-56d27115b82c9961.rlib" "/root/.rustup/toolchains/stable-x86_64-unknown-linux-musl/lib/rustlib/x86_64-apple-darwin/lib/libcompiler_builtins-8b28a0a374c38504.rlib" "-lSystem" "-lresolv" "-lc" "-lm" "-liconv" "-L" "/root/.rustup/toolchains/stable-x86_64-unknown-linux-musl/lib/rustlib/x86_64-apple-darwin/lib" "-o" "/tmp/cargobuild/x86_64-apple-darwin/release/deps/hello_cargo-849bb93aad9942c1" "-Wl,-dead_strip" "-nodefaultlibs"
#31 57.32 #     = note: clang-13: warning: no such sysroot directory: '/xx-sdk/MacOSX11.1.sdk' [-Wmissing-sysroot]
#31 57.32 #             ld: library not found for -lSystem
#31 57.32 #             clang-13: error: linker command failed with exit code 1 (use -v to see invocation)
#31 57.32 #
#31 57.32 #
#31 57.32 #   error: could not compile `hello_cargo` due to previous error
#31 57.32 #
#31 57.32 #   Caused by:
#31 57.32 #     process didn't exit successfully: `rustc --crate-name hello_cargo --edition=2018 src/main.rs --error-format=json --json=diagnostic-rendered-ansi,artifacts,future-incompat --crate-type bin --emit=dep-info,link -C opt-level=3 -C embed-bitcode=no -C metadata=849bb93aad9942c1 -C extra-filename=-849bb93aad9942c1 --out-dir /tmp/cargobuild/x86_64-apple-darwin/release/deps --target x86_64-apple-darwin -C linker=x86_64-apple-darwin-clang -L dependency=/tmp/cargobuild/x86_64-apple-darwin/release/deps -L dependency=/tmp/cargobuild/release/deps` (exit status: 1)
```

Also found out that the target triple is not correct for darwin and windows:

```
#31 48.22 not ok 9 darwin-amd64-hellocargo-rustup
#31 48.22 # (from function `assert_success' in file bats-assert/src/assert.bash, line 114,
#31 48.22 #  from function `testHelloCargo' in file test-cargo.bats, line 32,
#31 48.22 #  from function `testHelloCargoRustup' in file test-cargo.bats, line 44,
#31 48.22 #  in test file test-cargo.bats, line 93)
#31 48.22 #   `testHelloCargoRustup' failed
#31 48.22 #
#31 48.22 # -- command failed --
#31 48.22 # status : 1
#31 48.22 # output (2 lines):
#31 48.22 #   error: toolchain 'stable-x86_64-unknown-linux-musl' does not contain component 'rust-std' for target 'x86_64-unknown-macos10.6'
#31 48.22 #   note: not all platforms have the standard library pre-compiled: https://doc.rust-lang.org/nightly/rustc/platform-support.html
#31 48.22 # --
#31 48.22 #
```

Should be `x86_64-apple-darwin` instead of `x86_64-unknown-macos10.6`. Will fix that for `xx-info`.